src/pages/gift-cards/index.tsx

```diff
@@ -213,16 +245,26 @@ export default function GiftCardsPage() {
                       price={card.price}
                       className={classes.card}
                       actions={
-                        <Button
-                          component="a"
-                          href={card.url}
-                          target="_blank"
-                          rel="noopener noreferrer"
-                          rightIcon={<IconExternalLink size={16} />}
-                          fullWidth
-                        >
-                          Buy Now
-                        </Button>
+                        selectedVendor.id === 'kinguin' ? (
+                          <Button
+                            component="a"
```

Comment:
I guess this needs to be NextLink since it's an internal link
```suggestion
                            component={NextLink}
```